Sources/SignalProducer.swift

```diff
@@ -1878,6 +1878,59 @@ extension SignalProducerProtocol {
 	}
 }
 
+extension SignalProducerProtocol where Value == Bool {
+	/// Create a producer that computes a logical NOT in the latest values of `self`.
+	///
+	/// - returns: A producer that emits the logical NOT results.
+	public var negated: SignalProducer<Value, Error> {
+		return self.lift { $0.negated }
+	}
+	
+	/// Create a producer that computes a logical AND between the latest values of `self`
+	/// and `producer`.
+	///
+	/// - parameters:
+	///   - producer: Producer to be combined with `self`.
+	///
+	/// - returns: A producer that emits the logical AND results.
+	public func and(_ producer: SignalProducer<Value, Error>) -> SignalProducer<Value, Error> {
+		return self.lift(Signal.and)(producer)
```

Comment:
Oh, one more thing: may you please use `liftLeft` to be consistent with `combineLatest`?